runironic-inspector.sh

```diff
@@ -1,15 +1,41 @@
 #!/usr/bin/bash
 
-CONFIG=/etc/ironic-inspector/inspector.conf
+CONFIG=/etc/ironic-inspector/ironic-inspector.conf
 
 . /bin/ironic-common.sh
 
 wait_for_interface_or_ip
 
 cp $CONFIG $CONFIG.orig
 
-crudini --set $CONFIG ironic endpoint_override http://$IRONIC_URL_HOST:6385
-crudini --set $CONFIG service_catalog endpoint_override http://$IRONIC_URL_HOST:5050
+if [ ! -z "$CERT_FILE" ] || [ ! -z "$KEY_FILE" ] || [ ! -z "$CACERT_FILE" ] || [ ! -z "$INSECURE" ]; then
+    crudini --merge $CONFIG <<EOF
+[DEFAULT]
+host = $IRONIC_IP
+[ironic]
+endpoint_override = https://${IRONIC_URL_HOST}:6385
+$([ ! -z "$CACERT_FILE" ] && echo "cafile = $CACERT_FILE")
+$([ ! -z "$INSECURE" ] && echo "insecure = $INSECURE" || echo "insecure = false")
```

Comment:
even when INSECURE is false the value is set right? is this condition correct?

Reply:
it's rather "if insecure is set then set it to the value, otherwise set it to false"